test/conftest.py

```diff
@@ -173,10 +179,22 @@ def fixture_ecr_image(docker_registry, docker_base_name, tag):
 
 
 @pytest.fixture(autouse=True)
-def skip_by_device_type(request, use_gpu, instance_type):
+def skip_by_device_type(request, use_gpu, instance_type, accelerator_type):
     is_gpu = use_gpu or instance_type[3] in ['g', 'p']
-    if (request.node.get_closest_marker('skip_gpu') and is_gpu) or \
-            (request.node.get_closest_marker('skip_cpu') and not is_gpu):
```

Comment:
Need confirmation that this change for the if-condition is correct.

Reply:
I tested empirically by running the GPU and CPU tests. The GPU test skips EIA and CPU, the CPU test skips EIA and GPU. The EIA test skips CPU and GPU

Reply:
To make things easier to reason about, I changed the markers from skip_x to x_test. So basically the annotation tells you when to run the test, rather than when to not run the test.

So for example the `test_mnist_distributed_cpu` function is annotated with `@pytest.mark.cpu_test` now, rather than `@pytest.mark.skip_gpu`

Reply:
Thanks for clarifying. I missed the rename of the marker before. I have examined it more carefully, and it looks correct now.